### PR TITLE
trigger github action notebook checks

### PR DESCRIPTION
**Pull request by systemshift**
As @karlhigley pointed out to me, github actions only run 2 checks for tests. But does not run the other 2 checks when changes are applied to notebooks.

### CODE REVIEW

File: .github/workflows/run-translation-tests.yml [hunk 1]
```diff
@@ -3,7 +3,9 @@ name: Run notebook translation tests
 on:
   push:
     paths:
-    - 'examples/tutorials/**'
+    - "examples/tutorials/**"
+  pull_request:
```
Comment (karlhigley) [inline .github/workflows/run-translation-tests.yml]: I think this `pull_request` section also needs to specify the paths.

Comment (systemshift) [reply .github/workflows/run-translation-tests.yml]: > Thanks for taking this on! Looks like it’s running the translation tests on this PR, which it shouldn’t. I think it just needs to specify the paths for both pushes and pull requests.

Yes, I removed the changes I made to the notebook and it still triggers the checks. Will try and change that.

Comment (systemshift) [reply .github/workflows/run-translation-tests.yml]: I will revert the last commit if everything is fine.

> Looks like it’s running the translation tests on this PR, which it shouldn’t.
`examples/tutorials/**` includes all files in tutorials. If you want only the notebook files then it should be `example/tutorials/*.ipynb`

Comment (karlhigley) [reply .github/workflows/run-translation-tests.yml]: I think we do want everything in that directory (recursively) to trigger the translation tests. Does `**` cover that? Do we need `**/*`?

**Comment by karlhigley** [comment]
It looks like there's something off with the notebook tests action. It's running all the tests (including the notebooks/translations), but should only run the notebook tests.

**Comment by systemshift** [comment]
> It looks like there's something off with the notebook tests action. It's running all the tests (including the notebooks/translations), but should only run the notebook tests.

running `pytest test test/notebooks/test_notebooks.py::test_notebooks_basic_translations` on my local machine also runs all tests. I don't think this has to do with CI, perhaps the actual test units we have call the main test suite in some way.

File: .github/workflows/run-translation-tests.yml (hunk 1 above)
Comment (systemshift) [reply .github/workflows/run-translation-tests.yml]: no, `**` should work and include everything in `tutorials/`

**Pull request merged**